ros_ws/src/car_control/include/dms_controller.h

```diff
@@ -1,16 +1,21 @@
 #pragma once
 
+#include "drive_mode.h"
 #include <ros/ros.h>
 
 #include <chrono>
-#include <std_msgs/Bool.h>
+#include <std_msgs/Int32.h>
 #include <std_msgs/Int64.h>
 
 constexpr const char* PARAMETER_DMS_CHECK_RATE = "dms_check_rate";
 constexpr const char* PARAMETER_DMS_EXPIRATION = "dms_expiration";
+constexpr const char* PARAMETER_MODE_OVERRIDE = "mode_override";
 
-constexpr const char* TOPIC_DMS_HEARTBEAT = "/input/dms_heartbeat";
-constexpr const char* TOPIC_UNLOCK_MOTOR = "/commands/unlock_motor";
+constexpr const char* TOPIC_HEARTBEAT_MANUAL = "/input/heartbeat_manual";
+constexpr const char* TOPIC_HEARTBEAT_AUTONOMOUS = "/input/heartbeat_autonomous";
+constexpr const char* TOPIC_DRIVE_MODE = "/commands/drive_mode";
+
+constexpr DriveMode NO_OVERRIDE = DriveMode::LOCKED;
```

Comment:
I found the usage of this `constexpr` in the source code quite confusion as I wasn't always sure whether  `NO_OVERRIDE` was a global constant, a local constant or an enumerator.

Reply:
What do you suggest I do instead?

Reply:
We could just remove this variable and use DriveMode::LOCKED in the dms_controller.cpp directly